packages/alpinejs/src/directives/x-text.js

```diff
@@ -1,14 +1,18 @@
 import { directive } from '../directives'
 import { mutateDom } from '../mutation'
 
-directive('text', (el, { expression }, { effect, evaluateLater }) => {
+directive('text', (el, { expression, modifiers }, { effect, evaluateLater }) => {
     let evaluate = evaluateLater(expression)
 
     effect(() => {
         evaluate(value => {
-            mutateDom(() => {
-                el.textContent = value
-            })
+            value = value ?? null;
+
+            if (!modifiers.includes('ignore-nullish') || value !== null) {
```

Comment:
```suggestion
            if (!modifiers.includes('ignore-nullish') || value != null) {
```

NIT

Reply:
I would rather explicitly coerce to `null` and be able to do a strict check. It makes it more clear what the intent is here.